ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -24,7 +24,7 @@
 import com.github.ambry.clustermap.MockClusterMap;
```

Comment:
do you need any tests to check that support for BlobID V1 is not broken ? Any special cases?

Reply:
This should be covered in `injectionAccountAndContainerForGetHeadDeleteBlobIdV1Test`